### PR TITLE
Timwentz/working with navx and pid

### PR DESCRIPTION
**Pull request by wentzt**

### CODE REVIEW

File: src/org/usfirst/frc/team1775/robot/OI.java [hunk 1]
```diff
@@ -4,19 +4,23 @@
...
 
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-	public static Joystick myJoystick;
-	public static Joystick myJoystick2;
+	public static Joystick driverJoystick;
+	public static Joystick mechanismJoystick;
```
Comment (clebio) [inline src/org/usfirst/frc/team1775/robot/OI.java:16]: 👍

File: src/org/usfirst/frc/team1775/robot/subsystems/LiftSubsystem.java [hunk 2]
```diff
@@ -5,13 +5,40 @@
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 public class LiftSubsystem extends Subsystem {
+	private static final double START_RAMP_TIME_MS = 500.0;
+	private static final double MAX_SPEED = 0.6;
+	private static final double LIFT_SPEED_DEADBAND = 0.1;
```
Comment (clebio) [inline src/org/usfirst/frc/team1775/robot/subsystems/LiftSubsystem.java:10]: 👍

File: src/org/usfirst/frc/team1775/robot/subsystems/MotorSubsystem.java [hunk 3]
```diff
@@ -6,20 +6,28 @@
...
 	double driveToDistancePidResult = 0;
+	double rotateToAnglePidResult = 0;
 
 	private PIDController driveToDistancePidController = new PIDController(0, 0, 0, (PIDSource) RobotMap.driveEncoder,
 			(value) -> {
 				driveToDistancePidResult = value;
 			}, 0.02);
+	
+	private PIDController rotateToAnglePidController = new PIDController(0 ,0 ,0 ,(PIDSource) RobotMap.gyro,
+			(value) ->  {
```
Comment (clebio) [inline src/org/usfirst/frc/team1775/robot/subsystems/MotorSubsystem.java]: nice.

**Review by clebio — APPROVED** [review]
lgtm :shipit:

File: src/org/usfirst/frc/team1775/robot/subsystems/LiftSubsystem.java (hunk 2 above)
Comment (augustus-thomas) [reply src/org/usfirst/frc/team1775/robot/subsystems/LiftSubsystem.java:10]: No more magical numbers. Yay!

**Review by augustus-thomas — APPROVED** [review]
I liked the integration of constants in the Lift Subsystem. As long as the names are coherent and logical, it could be relatively simple to continue naming and assigning constants to the other subsystems. Let's make that a goal for this upcoming week.

File: src/org/usfirst/frc/team1775/robot/RobotMap.java [hunk 4]
```diff
@@ -18,22 +24,27 @@
 	public static Talon intakeMotorController1;
 	public static Talon intakeMotorController2;
 	public static Talon liftMotorController1;
-	public static Talon liftMotorController2;
+	public static DigitalInput liftBottomLimitSwitch;
+	public static DigitalInput liftTopLimitSwitch;
+	public static AHRS gyro;
 	
 	public static void init(){
 		driveEncoder = new Encoder(3, 2, false, Encoder.EncodingType.k1X);
```
Comment (gkd2000) [inline src/org/usfirst/frc/team1775/robot/RobotMap.java]: Oh man there are magic numbers here Gus won't be happy

File: src/org/usfirst/frc/team1775/robot/RobotMap.java [hunk 5]
```diff
@@ -18,22 +24,27 @@
...
+	    leftDriveMotorController = new Talon(0);
+	    rightDriveMotorController = new Talon(1);
 	    intakeMotorController1 = new Talon(3);
 	    intakeMotorController2 = new Talon(4);
 	    intakeMotorController2.setInverted(true);
 	    liftMotorController1 = new Talon(5);
-	    liftMotorController2 = new Talon(6);
+	    liftMotorController1.setInverted(true);
+	    liftBottomLimitSwitch = new DigitalInput(0);
+	    liftTopLimitSwitch = new DigitalInput(1);
```
Comment (gkd2000) [inline src/org/usfirst/frc/team1775/robot/RobotMap.java:54]: Should we change the numbers for the ports? or is that not worth it?

**Pull request merged**